packaging/hipify-clang.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(hipify-clang)
+
+install(PROGRAMS @PROJECT_BINARY_DIR@/hipify-clang/hipify-clang DESTINATION bin)
```

Comment:
If a common practice is to duplicate installation steps, those steps should be taken from https://github.com/ROCm-Developer-Tools/HIP/blob/master/hipify-clang/CMakeLists.txt#L83-L115. 

What about clang header files, being installed with hipify-clang?

Btw, could you test packaging without this particular line?